### PR TITLE
Ui/custom grid

### PR DESCRIPTION
**Pull request by YuhsinLiao1995**
## Description
Created a custom grid component to adapt our global theme setting.

## Checklist
- [v] Ran Linting
- [v] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [v] Wrote unit tests.
- [v] Added an entry to the `CHANGELOG.md` file.
- [v] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/grid/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      desktop: 1080,
+      maxWidth: 1100,
+    },
+  },
+};
+
+export const theme = createMuiTheme(common);
+describe('Grid', () => {
+  it('Works with base props', () => {
+    // const { classes } = useGetStyles();
```
Comment (ryuash) [inline src/grid/index.test.tsx]: remove

File: src/index.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {
 import ButtonExample from './button_example';
 import DataBlock from './data_block';
 import Footer from './footer';
+import StyledGrid from './grid';
```
Comment (ryuash) [inline src/index.ts]: why not just `import Grid from './grid';`

File: src/index.ts [hunk 3]
```diff
@@ -23,5 +24,6 @@ export {
   TwitterIcon,
   YoutubeIcon,
   Footer,
+  StyledGrid as Grid,
```
Comment (ryuash) [inline src/index.ts]: Then here you can just `Grid`

**Review by ryuash — CHANGES_REQUESTED** [review]
Just general clean up. Make sure to open a draft pr and review your own file changes before opening the PR and assigning me 👍

File: src/grid/index.test.tsx (hunk 1 above)
Comment (YuhsinLiao1995) [reply src/grid/index.test.tsx]: done

File: src/index.ts (hunk 2 above)
Comment (YuhsinLiao1995) [reply src/index.ts]: done

File: src/index.ts (hunk 3 above)
Comment (YuhsinLiao1995) [reply src/index.ts]: done

**Review by ryuash — APPROVED** [review]
Great!

**Pull request merged**